chainer/variable.py

```diff
@@ -709,6 +709,7 @@ def to_gpu(self, device=None):
         if self.data is None:
             self._initial_device = (cuda.Device().id
                                     if device is None else device)
+            self._data = [None]
```

Comment:
Could you add a short comment on this line, like `# renew the placeholder`? I could not understand this line at first sight.

Reply:
Okay, and I will add a thin object that represents initialized/uninitialized data, for which I use list, as another PR.

Reply:
Added a comment!